fuzzers/forkserver_simple/src/main.rs

```diff
@@ -96,12 +96,16 @@ pub fn main() {
     let mut fuzzer = StdFuzzer::new(scheduler, feedback, objective);
 
     // Create the executor for the forkserver
-    let mut executor = ForkserverExecutor::new(
-        "../../libafl_tests/src/forkserver_test.o".to_string(),
-        &[],
-        tuple_list!(edges_observer, time_observer),
+    let mut executor = TimeoutForkserverExecutor::new(
+        ForkserverExecutor::new(
+            "../../libafl_tests/src/forkserver_test.o".to_string(),
```

Comment:
This solution still feels a bit hackish in general :) the `.o` file should never end up in the `src` folder but somewhere in some `target/release`, but fine for now, thx